browser/app/js/browser/Login.js

```diff
@@ -139,13 +140,24 @@ export class Login extends React.Component {
               <i className="fas fa-sign-in-alt" />
             </button>
           </form>
-          {this.state.discoveryDoc &&
-            this.state.discoveryDoc.authorization_endpoint && (
+          {showOpenID && (
             <div className="openid-login">
               <div className="or">or</div>
-              <a href={"/login/openid"} className="btn openid-btn">
-                Log in with OpenID
-              </a>
+              {
+                this.state.clientId ? (
+                  <OpenIDLoginButton
+                    className="btn openid-btn"
```

Comment:
The option here seems to look fainter than the option in line 156, we can fix this by adding the line `color: @link-color;` to browser/app/less/inc/login.less:103 to ensure styles are consistent for the `openid-btn` class.

Here is the button when `clientId` is predefined:

<img width="781" alt="Screen Shot 2019-11-27 at 7 33 55 PM" src="https://user-images.githubusercontent.com/27314853/69775342-4c640a00-114d-11ea-902c-19f8b11e4126.png">

Here is the button when `clientId` is not predefined:

<img width="797" alt="Screen Shot 2019-11-27 at 7 30 04 PM" src="https://user-images.githubusercontent.com/27314853/69775362-5f76da00-114d-11ea-9970-bbf03123857b.png">